Formula/okta-aws-cli.rb

```diff
@@ -0,0 +1,22 @@
+class OktaAwsCli < Formula
+  desc "Okta authentication in support of aws cli operation"
+  homepage "https://github.com/okta/okta-aws-cli"
+  url "https://github.com/okta/okta-aws-cli/archive/refs/tags/v0.2.1.tar.gz"
+  sha256 "06374f0eb3e371a8ef1a5e8fb2bcd0e5bfcac607aa2083f6c1101b54713a47bf"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/okta-aws-cli"
+  end
+
+  test do
+    str_help = shell_output("#{bin}/okta-aws-cli --help")
+    assert_match "Usage::", str_help
```

Comment:
I noticed this extra colon in the actual output too, but it looks rather like a typo that will get changed sometime (https://github.com/okta/okta-aws-cli/pull/76).

```suggestion
    assert_match "Usage:", str_help
```

Reply:
Do change this one please.